<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QWidget" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="background">
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="user_input_line"/>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="password_input_line">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="login_button">
      <property name="text">
       <string>&amp;Login</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="password_label">
      <property name="text">
       <string>&amp;Senha:</string>
      </property>
      <property name="buddy">
       <cstring>password_input_line</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>&amp;Cancelar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="ip_label">
      <property name="text">
       <string>IP Servidor:</string>
      </property>
      <property name="buddy">
       <cstring>ip_input_line</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="user_label">
      <property name="text">
       <string>&amp;Usu√°rio:</string>
      </property>
      <property name="buddy">
       <cstring>user_input_line</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="port_label">
      <property name="text">
       <string>Porta Servidor:</string>
      </property>
      <property name="buddy">
       <cstring>port_input_line</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="ip_input_line"/>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="port_input_line"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>user_input_line</tabstop>
  <tabstop>password_input_line</tabstop>
  <tabstop>ip_input_line</tabstop>
  <tabstop>port_input_line</tabstop>
  <tabstop>cancel_button</tabstop>
  <tabstop>login_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
